extractor/src/main/java/org/schabi/newpipe/extractor/services/peertube/extractors/PeertubeChannelExtractor.java

```diff
@@ -40,18 +41,18 @@ public PeertubeChannelExtractor(StreamingService service, ListLinkHandler linkHa
     }
 
     @Override
-    public String getAvatarUrl() throws ParsingException {
+    public Image getAvatar() throws ParsingException {
         String value;
         try {
             value = JsonUtils.getString(json, "avatar.path");
         } catch (Exception e) {
             value = "/client/assets/images/default-avatar.png";
         }
-        return baseUrl + value;
+        return new Image(baseUrl + value, -1, -1);
```

Comment:
I know, you did not take a look at the other services yet, but it might be good to add a constant to the image class to make handling unknown dimensions easier and more readable.

Reply:
I wanted to make a constant(s?) for that, yeah. E.g. SoundCloud also provides multiple resolutions, but I think we can't know their resolutions. So maybe a SMALL and a LARGE constant?

Reply:
Yeah. UNKNOWN, SMALL, MEDIUM and LARGE sound good. Maybe add a `getSize()` method to the Image class which decides which images is small, medium or large.